app/routes/events/components/EventAdministrate/Attendees.tsx

```diff
@@ -156,14 +156,18 @@ const Attendees = ({
               Last ned
             </a>
           ) : (
-            <ConfirmModalWithParent
+            <ConfirmModal
               title="Eksporter til csv"
               closeOnConfirm={true}
               message={exportInfoMessage}
               onConfirm={createInfoCSV}
             >
-              <Button>Eksporter deltakere til csv</Button>
-            </ConfirmModalWithParent>
+              {({ openConfirmModal }) => (
+                <Button onClick={openConfirmModal}>
+                  Eksporter deltakere til csv
+                </Button>
+              )}
+            </ConfirmModal>
```

Comment:
Because there is some important information about exporting registrations that is detailed in the modal. So it emphasizes that the user known what he is doing and agrees to the terms.

Reply:
Ye that makes sense. For some reason I thought this was for exporting company interests to csv hehe